scripts/local_config.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
This shouldn't be a script, unless it actually does something when you invoke it ;-).